package.json

```diff
@@ -48,6 +48,6 @@
     "fs-extra": "^0.11.0",
     "lodash": "~2.4.1",
     "recursive-readdir": "^1.2.0",
-    "titanium": ">3.2.0"
```

Comment:
The version needs to be bumped up. If left as `">3.2.0"` grunt-titanium seems to use the old CLI, which itself requires users to log in. Thus `grunt` still fails with an `[ERROR] Authentication required` message. By setting this to `">4.0.0"` it works just fine with Fokke's other changes. The need to log in was removed in the 4.0 CLI.